python/tests/unit/connectors/ai/ollama/services/test_utils.py

```diff
@@ -0,0 +1,255 @@
+# Copyright (c) Microsoft. All rights reserved.
+from unittest.mock import MagicMock, patch
+
+import pytest
+
+from semantic_kernel.connectors.ai.function_call_choice_configuration import FunctionCallChoiceConfiguration
+from semantic_kernel.connectors.ai.function_choice_behavior import FunctionChoiceType
+
+# The code under test
+from semantic_kernel.connectors.ai.ollama.services.utils import (
+    MESSAGE_CONVERTERS,
+    update_settings_from_function_choice_configuration,
+)
+from semantic_kernel.connectors.ai.prompt_execution_settings import PromptExecutionSettings
+from semantic_kernel.contents.chat_message_content import ChatMessageContent
+from semantic_kernel.contents.function_call_content import FunctionCallContent
+from semantic_kernel.contents.function_result_content import FunctionResultContent
+from semantic_kernel.contents.image_content import ImageContent
+from semantic_kernel.contents.utils.author_role import AuthorRole
+
+
+@pytest.fixture
+def mock_chat_message_content() -> ChatMessageContent:
+    """Fixture to create a basic ChatMessageContent object with role=USER and simple text content."""
+    return ChatMessageContent(
+        role=AuthorRole.USER,
+        content="Hello, I am a user message.",  # The text content
+    )
+
+
+@pytest.fixture
+def mock_system_message_content() -> ChatMessageContent:
+    """Fixture to create a ChatMessageContent object with role=SYSTEM."""
+    return ChatMessageContent(role=AuthorRole.SYSTEM, content="This is a system message.")
+
+
+@pytest.fixture
+def mock_assistant_message_content() -> ChatMessageContent:
+    """Fixture to create a ChatMessageContent object with role=ASSISTANT."""
+    return ChatMessageContent(role=AuthorRole.ASSISTANT, content="This is an assistant message.")
+
+
+@pytest.fixture
+def mock_tool_message_content() -> ChatMessageContent:
+    """Fixture to create a ChatMessageContent object with role=TOOL."""
+    return ChatMessageContent(role=AuthorRole.TOOL, content="This is a tool message.")
+
+
+def test_message_converters_system(mock_system_message_content: ChatMessageContent) -> None:
+    """Test that passing a system message returns the correct dictionary structure for 'system' role."""
+    # Act
+    converter = MESSAGE_CONVERTERS[AuthorRole.SYSTEM]
+    result = converter(mock_system_message_content)
+
+    # Assert
+    assert result["role"] == "system", "Expected role to be 'system' on the returned message."
+    assert result["content"] == "This is a system message.", "Expected content to match the system message content."
```

Comment:
nit:
```suggestion
    assert result["content"] == mock_system_message_content.content, "Expected content to match the system message content."
```
This can avoid accidental changes to the message body that cause the tests to fail.

Reply:
Same for the other ones.

Reply:
Sure thing, will update the tests :).

Reply:
Tests updated to address this.